### PR TITLE
[webview] Fix comments (accidentially mixed `//` with `///`)

### PR DESCRIPTION
**Pull request by fzyzcjy**
The fix is self-explainable ;) Just two lines of doc.

*List which issues are fixed by this PR. You must list at least one issue.*

*If you had to change anything in the [flutter/tests] repo, include a link to the migration guide as per the [breaking change policy].*

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran [the auto-formatter]. (Unlike the flutter/flutter repo, the flutter/plugins repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. `[shared_preferences]`
- [ ] I listed at least one issue that this PR fixes in the description above.
- [x] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [x] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [ ] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[relevant style guides]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[exempt from version changes]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates
[following repository CHANGELOG style]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style
[the auto-formatter]: https://github.com/flutter/plugins/blob/master/script/tool/README.md#format-code
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests

### CODE REVIEW

**Comment by fzyzcjy** [comment]
Do I really need to create an issue for *this*? If so I will do it...

**Comment by fzyzcjy** [comment]
Done

**Comment by stuartmorgan** [comment]
When you've resolved the issues with the `publishable` check, please mark this as ready for review.

File: packages/webview_flutter/webview_flutter/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
-## NEXT
+## 3.0.1
 
 * Removes a duplicate Android-specific integration test.
 * Fixes an integration test race condition.
+* Fix comments (accidentially mixed // with ///).
```
Comment (stuartmorgan) [inline packages/webview_flutter/webview_flutter/CHANGELOG.md:5]: "Fixes". (Per the style guide linked from the PR checklist.)

**Review by stuartmorgan — CHANGES_REQUESTED** [review]

File: packages/webview_flutter/webview_flutter/CHANGELOG.md (hunk 1 above)
Comment (fzyzcjy) [reply packages/webview_flutter/webview_flutter/CHANGELOG.md]: Done

**Review by stuartmorgan — APPROVED** [review]

File: packages/webview_flutter/webview_flutter/lib/src/webview.dart [hunk 2]
```diff
@@ -715,8 +715,8 @@ class WebViewController {
   /// The Future completes with an error if a JavaScript error occurred.
   ///
   /// When running JavaScript in a [WebView], it is best practice to wait for
-  //  the [WebView.onPageFinished] callback. This guarantees all the JavaScript
-  //  embedded in the main frame HTML has been loaded.
+  ///  the [WebView.onPageFinished] callback. This guarantees all the JavaScript
+  ///  embedded in the main frame HTML has been loaded.
```
Comment (bparrishMines) [inline packages/webview_flutter/webview_flutter/lib/src/webview.dart:719]: nit: the spacing is slightly off

```suggestion
  /// the [WebView.onPageFinished] callback. This guarantees all the JavaScript
  /// embedded in the main frame HTML has been loaded.
```

**Review by bparrishMines — CHANGES_REQUESTED** [review]

**Review by bparrishMines — APPROVED** [review]
LGTM

**Pull request merged**